src/lib/alloc.c

```diff
@@ -601,7 +601,7 @@ void alloc_trace_buffer_heap(uint32_t caps, size_t bytes)
 	/* check buffer heap for capabilities */
 	trace_mem_init("heap: using buffer");
```

Comment:
What is the problem here? Please add the info in the commit message.

Reply:
Do you mean the additional include? Both API sets are still implemented in a single file. Moving this as I described in other comment could be a good next step while this patch addresses just the interface definition cleanup as a preparation for more readable user-level alloc.h for component developers and in general application level developers.

Reply:
I mean why use SOF_MEM_ZONE_BUFFER instead of RZONE_BUFFER a small explanation in the commit message wouldn't hurt.

Reply:
Commit message looks good now.